tests/builder/arg_name.rs

```diff
@@ -0,0 +1,14 @@
+use clap::{arg, Arg, Command};
```

Comment:
Could you move the tests to `tests/macros.rs`?  Thats where all the existing ones are

Reply:
sure. thanks for pointing that out.